scripts/docker/polkadot/polkadot_builder.Dockerfile

```
@@ -0,0 +1,40 @@
+FROM docker.io/paritytech/ci-linux:production as builder
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="builder" \
+	io.parity.image.description="This is the build stage for Polkadot. Here we create the binary." \
+	io.parity.image.source="https://github.com/paritytech/polkadot/blob/master/scripts/docker/polkadot/polkadot_builder.Dockerfile" \
+	io.parity.image.documentation="https://github.com/paritytech/polkadot/scripts/docker/polkadot/README.md"
+
+ARG PROFILE=release
+WORKDIR /polkadot
+
+COPY . /polkadot
+
+RUN cargo build --$PROFILE
+
+# ===== SECOND STAGE ======
+
+FROM docker.io/library/ubuntu:20.04
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="builder" \
+	io.parity.image.description="This is the 2nd stage: a very small image where we copy the Polkadot binary." \
```

The part:
```
This is the 2nd stage: a very small image where we copy the Polkadot binary.
```
can be kept as comment but this does not really describe the image.
```suggestion
	io.parity.image.description="Polkadot: a platform for web3. This is a self-buit multistage image." \
```